src/components/Jobs/components/RunningJobs/RunningJobs.js

```diff
@@ -0,0 +1,38 @@
+import React, { PureComponent } from 'react';
+import { FormattedMessage } from 'react-intl';
+import {
+  get,
+  filter,
+} from 'lodash';
+
+import JobsList from '../JobsList';
+import sortRunningJobs from './sortRunningJobs';
+import { DataFetcherContext } from '../../../DataFetcher/DataFetcherContext';
+import { RUNNING } from '../../jobStatuses';
```

Comment:
Please follow importing order convention. From more far imports to the more closer.

Reply:
import { DataFetcherContext } from '../../../DataFetcher/DataFetcherContext';
import { RUNNING } from '../../jobStatuses';
import JobsList from '../JobsList';
import sortRunningJobs from './sortRunningJobs';

Should it be like that? Is there any docs or something where I can read about that?

Reply:
Yes.